### ISSUE TITLE
Error: A dynamic link library (DLL) initialization routine failed.

### ISSUE THREAD

**Issue by cespon** [tensorflow/tfjs#3634]
### Software version

**TensorFlow.js**: `@tensorflow/tfjs-node@2.0.1`
**OS**: `Windows 10 Pro 64 bit`
**NodeJS**: `v12.8.2`

### Description of the problem

When I try to execute a program that uses `tfjs-node`, I got the following error:

```
Overriding the gradient for 'Max'
Overriding the gradient for 'OneHot'
Overriding the gradient for 'PadV2'
Overriding the gradient for 'SpaceToBatchND'
Overriding the gradient for 'SplitV'
internal/modules/cjs/loader.js:1188
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: A dynamic link library (DLL) initialization routine failed.
\\?\C:\Users\SERVER\tftest\node_modules\@tensorflow\tfjs-node\lib\napi-v6\tfjs_binding.node
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1188:18)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\SERVER\tftest\node_modules\@tensorflow\tfjs-node\dist\index.js:58:16)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
```

The folder `C:\Users\SERVER\tftest\node_modules\@tensorflow\tfjs-node\lib\napi-v6` has two files inside:

- `tensorflow.dll`
- `tfjs_binding.node`

The same program works great in my development computer, witch has the same software described (only different hardware). After several tests **I think that this issue is related with the CPU model used** in my "production server". I triying to execute this program in a low consumption PC that uses a Intel Celeron J3455 (instructions supported: MMX, SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2, EM64T, VT-x, AES, SHA) while my development PC has a Intel Core i7-7700HQ.

I tried with multiple `@tensorflow/tfjs-node` versions and NodeJS versions without success.

### Code to reproduce the bug 

We can reproduce the issue with a `app.js` with the TensorFlow require call only. At the beginning, the folder `C:\Users\SERVER\tftest` has only two files:

#### `package.json`
```
{
  "name": "tensorflow-test",
  "version": "1.0.0",
  "main": "app.js",
  "dependencies": {
    "@tensorflow/tfjs-node": "^2.0.1"
  }
}
```

#### `app.js`
```
const tensorflow = require('@tensorflow/tfjs-node')
```

After `npm install` execution, a `node_modules` folder and `package-lock.json` file were created.

If we execute `node app.js`, the error described above is shown.

**Comment by rthadur** [comment]
@cespon can you please check related [issue](https://github.com/tensorflow/tfjs/issues/2619#issuecomment-575347028) here ?

**Comment by kvntzn** [comment]
@cespon were you able to resolve the issue ?

**Comment by cespon** [comment]
I couldn't find a native solution for NodeJS. Intel Celeron J3455 doesn't support AVX instructions, so I searched for a solution for no-AVX processors. After some days searching, I found [tensorflow-windows-wheel](https://github.com/fo40225/tensorflow-windows-wheel) for Python. 

I use [this wheel](https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.15.0/py37/CPU%2BGPU/cuda101cudnn76sse2) with Python 3.7 and I call a Python script from NodeJS when I need to use TensorFlow. It is the only solution I have found. If you know any other, let me know.
